Makefile

```diff
@@ -183,12 +183,12 @@ all: clean resources $(BOOT_ETH_SUBDIRS) cromsubdirs xromwell.xbe vmlboot $(BOOT
 ifeq ($(ETHERBOOT), yes)
 ethsubdirs: $(patsubst %, _dir_%, $(ETH_SUBDIRS))
 $(patsubst %, _dir_%, $(ETH_SUBDIRS)) : dummy
-	$(MAKE) CFLAGS="$(ETH_CFLAGS)" -C $(patsubst _dir_%, %, $@)
+	@$(MAKE) CFLAGS="$(ETH_CFLAGS)" -C $(patsubst _dir_%, %, $@)
```

Comment:
We probably still want something like this, so we can debug issues in the build-scripts without having to change a lot of the statements:

https://github.com/XboxDev/nxdk/blob/b07ca9ae1f2f37dcda583fac34a832f8260630bb/Makefile#L107-L116

`V` is what GNU autotools typically use, and `VERBOSE` is what CMake typically uses.

There's also `make -n` but it won't actually run commands or show output.

Reply:
I'm going to take this as a general commit and not specific to this line (as it would only apply to the ethboot module). This PR is very low time investment for me and I'm uninterested in providing large changes.